Cargo.toml

```diff
@@ -22,6 +22,7 @@ members = [
     "solana",
     "task_manager",
 ]
+resolver = "2"
```

Comment:
the changes in this file seem unneeded

Reply:
The latest rust (1.72.0) has started throwing some noisy warnings/notes on compile without this

Reply:
how incredibly annoying that they just randomly started requiring explicitly using a next gen resolver.. oh well

Reply:
They didn’t, just required you explicitly declare one. If this should “1” that’s cool too